src/cpu/x64/jit_avx512_core_amx_1x1_conv_kernel.cpp

```diff
@@ -170,19 +170,19 @@ void jit_avx512_core_amx_1x1_fwd_kernel_t::interleave_store() {
 }
 
 Ymm jit_avx512_core_amx_1x1_fwd_kernel_t::ymm_mask(
-        const Ymm &ymm_in, bool mask_flag, bool store) {
+        const Ymm ymm_in, bool mask_flag, bool store) {
```

Comment:
Please update commit message to use the format `src: cpu: x64: replace...`

Reply:
Done